### ISSUE TITLE
False positive for mac.com

### ISSUE THREAD

**Issue by benrugg** [ZooTools/email-spell-checker#25]
## False positive: anything@mac.com suggests anything@mail.com

1. How popular is the TLDs or Domain: `mac.com` email addresses were issued to millions of people. [Apple has moved away from them, but they still work](https://appleinsider.com/articles/20/08/31/twelve-years-later-apple-is-still-trying-to-erase-maccom-email-addresses). 
2. Why most of the companies using this package will get benefit of it: False positives for a large number of email addresses cause additional customer support costs.

## Current Behavior

```
const suggestion = emailSpellChecker.run({ email: 'whatever@mac.com' })
suggestion.full == 'whatever@mail.com'
```

## Expected Behavior

```
const suggestion = emailSpellChecker.run({ email: 'whatever@mac.com' })
suggestion == null
```

## Steps to Reproduce the Problem

1. See above

**Comment by benrugg** [comment]
Thank you for this package, btw. It's really helpful! I'll try to submit a pr for this when I have a second.
